tools/dependency/generate_external_dep_rst.py

```diff
@@ -3,10 +3,16 @@
 # Generate RST lists of external dependencies.
 
 from collections import defaultdict, namedtuple
+import os
 import pathlib
 import sys
 import urllib.parse
 
+# TODO(phlax): remove this
+# this needs to be before the utils import
+# if we move cve_scan to a bazel job then we can remove
+os.environ["API_PATH"] = "external/envoy_api_canonical"
```

Comment:
Oh, I see. How complex is it to make Bazelify cve_scan?

Reply:
not too complex i think

would you prefer to update that first ?

Reply:
pr added for `cve_scan` here #15342

Reply:
and merged with this PR to simplify...